arkouda/groupbyclass.py

```diff
@@ -263,9 +263,12 @@ def __init__(
                 self.keys, return_groups=True, assume_sorted=self.assume_sorted
             )
 
-        self.size = self.permutation.size
+        self.length = self.permutation.size
         self.ngroups = self.segments.size
 
+    def size(self):
+        return self.count()
```

Comment:
We need to add a docstring to this function. It should indicate that this alias was added to conform with Pandas API.

Reply:
Done

Reply:
The docstring at a minimum should also have a `See Also` section that points lists count. I would also add a Returns for users using `Groupby.size?` in ipython to view docstring, just makes it easier for them to know what to expect.

Reply:
Done